lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -0,0 +1,938 @@
+
+  get '/' => sub ($c) {
+    $c->render(text => 'Hello World!');
+  };
+
+  app->start;
+
+With L<Mojolicious::Command::Author::generate::lite_app> there is also a helper command to generate a small example
+application.
+
+  $ mojo generate lite-app myapp.pl
+
+=head2 Commands
+
+Many different L<commands|Mojolicious::Commands/"COMMANDS"> are automatically available from the command line. CGI and
+L<PSGI> environments can even be detected and will usually just work without commands.
+
+  $ ./myapp.pl daemon
+  Web application available at http://127.0.0.1:3000
+
+  $ ./myapp.pl daemon -l http://*:8080
+  Web application available at http://127.0.0.1:8080
+
+  $ ./myapp.pl cgi
+  ...CGI output...
+
+  $ ./myapp.pl get /
+  Hello World!
+
+  $ ./myapp.pl
+  ...List of available commands (or automatically detected environment)...
+
+A call to L<Mojolicious/"start"> (C<app-E<gt>start>), which starts the command system, should be the last expression in
+your application, because its return value can be significant.
+
+  # Use @ARGV to pick a command
+  app->start;
+
+  # Start the "daemon" command
+  app->start('daemon', '-l', 'http://*:8080');
+
+=head2 Reloading
+
+Your application will automatically reload itself if you start it with the L<morbo> development web server, so you
+don't have to restart the server after every change.
+
+  $ morbo ./myapp.pl
+  Web application available at http://127.0.0.1:3000
+
+For more information about how to deploy your application see also L<Mojolicious::Guides::Cookbook/"DEPLOYMENT">.
+
+=head2 Routes
+
+Routes are basically just fancy paths that can contain different kinds of placeholders and usually lead to an action,
+if they match the path part of the request URL. The first argument passed to all actions (C<$c>) is a
+L<Mojolicious::Controller> object, containing both the HTTP request and response.
+
+  use Mojolicious::Lite -signatures;
+
+  # Route leading to an action that renders some text
+  get '/foo' => sub ($c) {
+    $c->render(text => 'Hello World!');
+  };
+
+  app->start;
+
+Response content is often generated by actions with L<Mojolicious::Controller/"render">, but more about that later.
+
+=head2 GET/POST parameters
+
+All C<GET> and C<POST> parameters sent with the request are accessible via L<Mojolicious::Controller/"param">.
+
+  use Mojolicious::Lite -signatures;
+
+  # /foo?user=sri
+  get '/foo' => sub ($c) {
+    my $user = $c->param('user');
+    $c->render(text => "Hello $user.");
+  };
+
+  app->start;
+
+=head2 Stash and templates
+
+The L<Mojolicious::Controller/"stash"> is used to pass data to templates, which can be inlined in the C<DATA> section.
+A few stash values like C<template>, C<text> and C<data> are reserved and will be used by
+L<Mojolicious::Controller/"render"> to decide how a response should be generated.
+
+  use Mojolicious::Lite -signatures;
+
+  # Route leading to an action that renders a template
+  get '/foo' => sub ($c) {
+    $c->stash(one => 23);
```

Comment:
It would be awesome to include `app->default({zero => 20, one => 21});` in this example too to introduce people to default.  I actually never use default because I've never seen it used in examples and is only documented once in `Mojolicious.pm`, so recognizing an option like this is even available can be tough (there's so much documentation one can't keep it all memorized).